src/efjm.tw2

```diff
@@ -5,6 +5,7 @@ Escape from Juggalo Mountain - twee2 Version
 Escape from Juggalo Mountain
 
 [[Begin|Jail Cell (Unlit) - name character]]
+[[Opening Crawl|Opening crawl]]
```

Comment:
Like we discussed, I think this is a fine convention to use until we're ready to ship the finished game. Honestly, we could even keep these links in and just hide them under a `$debug` variable that's false by default. Could be handy to have these available if we need to go back later and change things.

Reply:
Yeah, I really like that debug idea. That would definitely speed up any kind of testing or last minute changes to the later sections of the game as we're finalizing.